### ISSUE TITLE
"Explore" page design issues

### ISSUE THREAD

**Issue by samueldr** [NixOS/nixos-homepage#651]
TODO:

 - [ ] Verify lambdaflake colours.
 - [ ] Verify other colours, _if_ they should match the palette or if deviation is desired.

_Originally posted by @samueldr in https://github.com/NixOS/nixos-homepage/issues/640#issuecomment-733929418_

**Comment by garbas** [comment]
I have uploaded the source file for the Explore page here https://github.com/NixOS/nixos-artwork/blob/master/www/2020_redesign/Features/Graphics.pdf

**Comment by samueldr** [comment]
Just saying, this isn't solved, even ignoring the new colours.

<img width="1426" height="733" alt="Image" src="https://github.com/user-attachments/assets/8b5d28e6-7157-425a-aab3-b5828fd340e5" />

The Lambdaflakes were either in whole or partly miscoloured. As is shown here on the rocket, two out of three of the darker blue lambdas have the gradient misapplied. As for the light lambdas, at least one is visibly incorrect.

The colours in the pictures also were off the (at the time) selected palette. Now that the selected palette was changed, it likely is even more.

**Comment by thilobillerbeck** [comment]
Yeah, good point, oversaw that matter while trying to clean up old issues. I'll reopen this then (even if we want to sunset explore in the long term)
